rs_scraper.py

```diff
@@ -1,17 +1,16 @@
 import requests, sqlite3
 from lxml import html
 
-class RS_Scraper:
+class RSScraper:
+
     def __init__(self, thread):
         #initialize scraping object
-        self.thread = thread
-        self.timestamps = []
-        self.bodies     = []
-        self.usernames  = []
+        self.thread = thread  #url of the thread to be scraped
```

Comment:
make this property name more clear and delete the comment since you don't need it if the code speaks for itself
```suggestion
        self.thread_url = thread_url
```